Dockerfile

```diff
@@ -28,7 +38,12 @@ RUN bundle install
 # Copy over the rest of the app
 COPY . .
 
+RUN ["chmod", "+x", "./docker-entrypoint.sh"]
+
 EXPOSE 3001
 
-CMD ["bundle", "exec", "rails", "assets:precompile"]
+ENTRYPOINT ["./docker-entrypoint.sh"]
+
+# As written this first command will not execute since it's the first of two `CMD`s
+CMD ["bundle", "exec", "rails", "assets:precompile"] 
```

Comment:
@aaronbrezel would you mind changing this to `RUN`. I have a clear memory of changing this a few weeks back, but I'm wondering if I packaged it with something that got reverted 🤔 this might have something to do with the delays on initial load

Reply:
nvm, I will change this in a fix PR!

Reply:
gotchu https://github.com/WPMedia/klaxon/pull/33